ForageBuddy/Form1.cs

```diff
@@ -11,33 +14,81 @@ public partial class Form1 : Form
         private string _folderPath;
         private List<string> _filesInScreenshotDirectory;
         private Dictionary<string, PlayerScore> _playerScores;
-
+        
+        private readonly ILogger _logger;
         private readonly ImageParser _imageParser;
+        private readonly string _logFile = "log.txt";
 
         public Form1()
         {
             _imageParser = new ImageParser();
             InitializeComponent();
-            lbScores.Items.Insert(0, "Please Specify Screenshot folder");
+            lbScores.Items.Insert(0, "Please specify a screenshot folder.");
+
+            var logFolder = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "logs");
+
+            // Check for logFolder, create if missing
+            if (!Directory.Exists(logFolder))
+                Directory.CreateDirectory(logFolder);
+
+            // Check for logFile, create new name if one already exists
+            if (File.Exists(Path.Combine(logFolder, _logFile)))
+            {
+                var i = 1;
+                
+                _logFile = _logFile.Insert(_logFile.IndexOf("."), $" ({i})");
+                
+                while (File.Exists(Path.Combine(logFolder, _logFile)))
+                {
+                    i++;
+                    _logFile = _logFile.Replace($"{i - 1}", $"{i}");
```

Comment:
The segment for Line 34 & on will make duplicates of the "log.txt" file into "log (1).txt" and so on for each launch of the program. Probably not the most efficient way though, but added it for show in case you might want it.